### PR TITLE
[FR] Creating new French language

### PR DESCRIPTION
**Pull request by Gastonia02**
Adding french setup for the translation

### CODE REVIEW

File: main_fr.tex [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  Version 1.0
+}
+
+\newcommand{\intro}{
+  This is a community-driven project, which has a \href{https://github.com/Heegu-sama/Homm3BG}{GitHub repository}.
+  Everyone is welcome to contribute, make changes, and fix errors.
+  If you simply want to leave feedback, please do so in the original \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{BoardGameGeek thread}.
+}
+
+\title{\includegraphics[width=6cm]{\images/title.png}\\The Rewrite}
```
Comment (qwrtln) [inline main_fr.tex:19]: Can you translate it straight away? We'll get the metadata out of the way.

**Comment by qwrtln** [comment]
Please follow the instructions I sent you on Discord, and let's carry on 👍🏻

File: main_fr.tex (hunk 1 above)
Comment (Gastonia02) [reply main_fr.tex:19]: Should I do it on my branch and submit a pull request ?

Comment (qwrtln) [reply main_fr.tex:19]: Just push it to your existing branch so that it's updated here.

File: main_fr.tex [hunk 2]
```diff
@@ -0,0 +1,21 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+% Language setting
+\usepackage[french]{babel}
```
Comment (qwrtln) [inline main_fr.tex:5]:
```suggestion
\usepackage[T1]{fontenc}
\usepackage[english]{babel}
```

File: main_fr.tex [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+\def\sections{sections}
```
Comment (qwrtln) [inline main_fr.tex:7]:
```suggestion
\def\sections{sections/translated/fr}
\def\lang_header_adjustment{0px}

```

File: main_fr.tex [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+\newcommand{\version}{
+  Version 1.0
```
Comment (qwrtln) [inline main_fr.tex:10]: Is this how you spell "Version" in French?

File: main_fr.tex [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+\newcommand{\intro}{
+  Ceci est un projet communautaire doté d'un \href{https://github.com/Heegu-sama/Homm3BG}{dépôt GitHub}.
+  Vous êtes tous invités à apporter vos contributions, modifications, corrections et traductions.
+  Si vous souhaitez simplement faire part de vos commentaires, merci de le faire dans le \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{fil de discussion BoardGameGeek} original.
+}
+
```
[the remark this answers is not in the document]
Comment (qwrtln) [reply main_fr.tex:18]: You will need this for the printable version.

File: README.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ Click in the table to download the most recent builds in the chosen language:
...
 | 🇪🇸 Español    |~5%|[🇪🇸📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/es/main_es.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/es/printable_es.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_es/components_list_es.pdf)|
-| 🇫🇷 Français   |0%|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_fr/components_list_fr.pdf)|
+| 🇫🇷 Français   |~5%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_fr/components_list_fr.pdf)|
```
Comment (qwrtln) [inline README.md:15]:
```suggestion
| 🇫🇷 Français   |~5%|[🇫🇷📜](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/main_fr.pdf)|[🇫🇷🖨️](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/fr/printable_fr.pdf)|[n/a](https://raw.githubusercontent.com/qwrtln/Homm3BG-build-artifacts/components_list_fr/components_list_fr.pdf)|
```

**Review by qwrtln — CHANGES_REQUESTED** [review]
Last pieces of the puzzle:
1. `.github/qr-codes-fr.tex`
2. RULE BOOK REWRITE in French

**Review by qwrtln — CHANGES_REQUESTED** [review]
Please leave the automatically generated `fr.po` files in this PR. You will add translations in PRs that follow.

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -25,9 +25,12 @@ po4a.cfg @ohgodnotagain @qwrtln @DarthGandalf
...
-/translations/assets/es/ @luischos @qwrtln @Heegu-sama
-main.es.tex @luischos @qwrtln @Heegu-sama
+**/es.po @luischos @qwrtln
+/translations/assets/es/ @luischos @qwrtln
+main_es.tex @luischos @qwrtln
 #
 # French
+**/fr.po @Gastonia02 @qwrtln
+/translations/assets/fr/ @Gastonia02 @qwrtln
+main_fr.tex @Gastonia02 @qwrtln
```
Comment (qwrtln) [inline .github/CODEOWNERS:35]:
```suggestion
**/es.po @luischos @qwrtln @Heegu-sama
/translations/assets/es/ @luischos @qwrtln @Heegu-sama
main_es.tex @luischos @qwrtln @Heegu-sama
#
# French
**/fr.po @Gastonia02 @qwrtln @Heegu-sama
/translations/assets/fr/ @Gastonia02 @qwrtln @Heegu-sama
main_fr.tex @Gastonia02 @qwrtln @Heegu-sama
```

**Review by qwrtln — APPROVED** [review]

**Pull request merged**